test/sharness/t0080-repo.sh

```diff
@@ -98,7 +104,7 @@ test_expect_success "remove recursive pin, add direct" '
 
 test_expect_success "remove direct pin" '
 	echo "unpinned $HASH" >expected6 &&
-	ipfs pin rm "$HASH" >actual6 &&
+	ipfs pin rm -r=false "$HASH" >actual6 &&
```

Comment:
This behaviour shouldnt change though, `ipfs pin rm` should remove either a recursive or a direct pin